### PR TITLE
feat(fedora): support fedora

### PR DESCRIPTION
**Pull request by maito1201**
refs. https://github.com/vulsio/goval-dictionary/pull/196 

Trailing white spaces are automatically removed by editor.

### CODE REVIEW

File: docker/update-all.sh [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
+./oval.sh --oracle ${@}
+./oval.sh --fedora ${@}
+./gost.sh --redhat --batch-size 500 ${@}
+./gost.sh --debian --batch-size 500 ${@}
+./gost.sh --ubuntu --batch-size 15 ${@}
+./nvd.sh ${@}
+./jvn.sh ${@}
+./exploitdb.sh ${@}
+./msfdb.sh ${@}
+./kev.sh ${@}
```
Comment (maito1201) [inline docker/update-all.sh:17]: image

Certainly Line feed code is LF, why...

**Review by MaineK00n — APPROVED** [review]

**Comment by MaineK00n** [comment]
After the PR to goval-dictinoary is merged, check the behavior again.

File: docker/update-all.sh (hunk 1 above)
Comment (MaineK00n) [reply docker/update-all.sh:17]: This command seems to be related to space.
```console
$ git diff support_fedora:docker/update-all.sh..master:docker/update-all.sh
```

File: docker/oval.sh [hunk 2]
```diff
@@ -49,10 +49,15 @@ case "$target" in
 		-v $PWD:/goval-dictionary \
 		vuls/goval-dictionary fetch oracle ${@}
 		;;
-	--*)  echo "specify [--redhat --amazon --debian --ubuntu --alpine --oracle]"
+	--fedora) docker run --rm -it \
+		${DOCKER_NETWORK_OPT} \
+		-v $PWD:/goval-dictionary \
+		vuls/goval-dictionary fetch fedora ${@} 33 34 35
```
Comment (MaineK00n) [inline docker/oval.sh:55]:
```suggestion
		vuls/goval-dictionary fetch fedora ${@} 32 33 34 35
```

File: install-host/oval.sh [hunk 3]
```diff
@@ -1,36 +1,39 @@
...
-	--alpine) 
+	--alpine)
 		goval-dictionary fetch alpine ${@} 3.2 3.3 3.4 3.5 3.6 3.7 3.8 3.9 3.10 3.11 3.12 3.13 3.14
 		;;
 	--oracle)
 		goval-dictionary fetch oracle ${@}
 		;;
-	--*)  echo "specify [--redhat --amazon --debian --ubuntu --alpine --oracle]"
+	--fedora)
+		goval-dictionary fetch fedora ${@} 33 34 35
```
Comment (MaineK00n) [inline install-host/oval.sh:31]:
```suggestion
		goval-dictionary fetch fedora ${@} 32 33 34 35
```

Comment (maito1201) [reply install-host/oval.sh:31]: then why  `redhat 5` and `debian 7 `,  `ubuntu 14` is not written?

Comment (maito1201) [reply install-host/oval.sh:31]: OS that is EOL is omitted. So Fedora 32 and 33 is prefer to omit.
https://endoflife.date/fedora

Comment (MaineK00n) [reply install-host/oval.sh]: I now understand the policy of removing EOL from fetch.

**Review by MaineK00n — APPROVED** [review]

**Pull request merged**